kubetest/util.go

```diff
@@ -138,9 +139,17 @@ func finishRunning(cmd *exec.Cmd) error {
 		return fmt.Errorf("error starting %v: %v", stepName, err)
 	}
 
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	finished := make(chan error)
 
 	go func() {
+		sig := <-sig_channel
+		log.Printf("Killing %v after receiving %s", -cmd.Process.Pid, sig)
+		if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL); err != nil {
+			fmt.Errorf("error killing %v: %v", -cmd.Process.Pid, err)
```

Comment:
I did dig some code and there are not error handling when calling syscall.Kill with syscall.SIGKILL like
https://github.com/kubernetes/test-infra/blob/d70d82c5a94fd24064a9ddaf08d099f1a932da22/kubetest/util.go#L152
https://github.com/kubernetes/kubernetes/blob/6858bb4e6025218b2f4013173dc027709819bc8c/vendor/github.com/opencontainers/runc/libcontainer/standard_init_linux.go#L150

So it would be fine without error handling here.